Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN adduser --uid 431 --disabled-password --disabled-login --gecos 'webdev' --no
 
 RUN apt-get update && \
```

Comment:
Let's also update the image from stretch to buster.

Reply:
:+1: